avocado/plugins/run.py

```diff
@@ -162,9 +162,11 @@ def run(self, args):
             sys.exit(exit_codes.AVOCADO_FAIL)
         job_instance = job.Job(args)
         job_run = job_instance.run()
-        result_dispatcher = ResultDispatcher()
-        if result_dispatcher.extensions:
-            result_dispatcher.map_method('render',
-                                         job_instance.result,
-                                         job_instance)
+
+        if job_instance.status != 'RUNNING':
```

Comment:
OK I played with it and this patch prevents things like this:

```
avocado run fsafdsfa

Unable to resolve reference(s) 'fsafdsfa' with plugins(s) 'file', 'external', try running 'avocado list -V fsafdsfa' to see the details.
JOB HTML   : /home/medic/avocado/job-results/job-2016-11-03T19.54-1a6f1da/html/results.html
```

But I still believe it should be treated inside the plugins as with this patch you have no way to react on it. But if we keep executing the `render` methods even on RUNNING jobs we have the greater flexibility for plugins.